templates/expo-template-bare-minimum/android/settings.gradle

```diff
@@ -4,7 +4,7 @@ pluginManagement {
 plugins { id("com.facebook.react.settings") }
 
 extensions.configure(com.facebook.react.ReactSettingsExtension) { ex ->
-  if (System.getenv('EXPO_UNSTABLE_CORE_AUTOLINKING') == '1') {
```

Comment:
we can remove `EXPO_UNSTABLE_CORE_AUTOLINKING` and the block entirely. given we don't have rnc-cli installed from 0.76

Reply:
Sure, gonna remove that for now but the idea of ENG-13703 is to allow users to opt out of core autolinking and use the community autolinker if they want

Reply:
cool then we should change the var name and document that people should install rnc-cli explicitly.

Reply:
yep, I've removed the variable for now and will add it back with a different name in a followup PR